.github/workflows/checks.yml

```diff
@@ -31,7 +31,9 @@ jobs:
       - name: Test Server RPM
         run: |
           export MAVEN_OPTS="${MAVEN_INSTALL_OPTS}"
-          ./bin/retry ./mvnw install -B -P ci -pl presto-server-rpm
+          ./bin/retry ./mvnw package -B -P ci -pl presto-server-rpm
```

Comment:
This needs to be the `verify` phase since that's when then integration tests run (see the `maven-failsafe-plugin` configuration)